ivy_tests/test_ivy/test_frontends/test_jax/test_jax_src_api.py

```diff
@@ -4,6 +4,83 @@
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 from ivy.functional.frontends.jax._src.api import device_put, device_get
+from hypothesis import given, assume, strategies as st
+
+
+def _fn1(x, y):
+    return ivy.matmul(x, y)
+
+
+def _fn2(x, y):
+    return ivy.vecdot(x, y)
+
+
+def _fn3(x, y):
+    ivy.add(x, y)
```

Comment:
Did you forget `return` or was it on purpose? 😅

Reply:
It was on purpose as this function used by vmap.

Reply:
oh, I forgot.